kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/expressions/LikeExpressionEvaluator.java

```diff
@@ -183,4 +183,18 @@ private static String escapeLikeRegex(String pattern, char escape) {
     }
     return "(?s)" + javaPattern;
   }
+
+  /** Escapes characters escapeChar in the input String */
+  static String escape(String input, char escapeChar) {
```

Comment:
is the `_` in the input be escaped as well? Otherwise it will be treated as LIKE `_` (match any single characters)?